packages/graphback-datasync/src/util.ts

```diff
@@ -39,3 +44,35 @@ export const DataSyncFieldNames = {
   lastUpdatedAt: '_lastUpdatedAt',
   deleted: '_deleted'
 }
+
+
+
+export interface DataSyncModelConflictConfig {
+  enabled: boolean
+  conflictResolution?: ConflictResolutionStrategy
+}
+
+export interface DataSyncModelConfigMap {
+  [modelName: string]: DataSyncModelConflictConfig
+}
+
+export interface ConflictResolutionStrategy {
+  resolveUpdate(conflict: ConflictMetadata): any
+}
+
+export const ServerSideConflictResolution: ConflictResolutionStrategy = {
+  resolveUpdate(conflict: ConflictMetadata): any {
+    const { serverDiff, clientDiff } = conflict;
+
+    const resolved: any = {};
+
+    Object.keys(clientDiff).forEach((fieldName: string) => {
+      if (!serverDiff[fieldName]) {
```

Comment:
We really need to have tests for this part. It is clearly from running this code that with this strategy we going to lose some data. 

Maybe you wanted to use full server snapshot and then apply client side diff on top of it? 
This would be something like client side first (without checking if conflict actually occurred. 

Second approach will be to merge automatically what is mergable and pass to conflictResolutionStrategy only fields that are conflicted - I have done that before but do not like as it limits flexibility of having full view of the doc.

Reply:
> Maybe you wanted to use full server snapshot and then apply client side diff on top of it?

To me:
serverDiff: What fields have changed on the server since client passed version _version
clientDiff: What fields the client wants to update
resolved: The fields that are in clientDiff but not in serverDiff

But I see that that was wrong so I will change it

Reply:
> resolved: The fields that are in clientDiff but not in serverDiff

Ok.. So we have problem with the interface. Maybe quick chat about this. 
I thought resolved is actual final object that will be saved. I see it now when debugging. 

Generally let's have quick chat about this approach - I never seen it but this actually might make a lot of sense.